tests/integration/update_cluster/privatecalico/kubernetes.tf

```diff
@@ -404,17 +405,19 @@ resource "aws_route_table" "private-us-test-1a-privatecalico-example-com" {
   vpc_id = "${aws_vpc.privatecalico-example-com.id}"
 
   tags = {
-    KubernetesCluster = "privatecalico.example.com"
-    Name              = "private-us-test-1a.privatecalico.example.com"
+    KubernetesCluster                                 = "privatecalico.example.com"
+    Name                                              = "privatecalico.example.com"
```

Comment:
@blakebarnett / @mikesplain notice the change in the `Name` tag.  Just want to verify that the change is correct.  Do we need the private-us-test-1a.privatecalico.example.com?

Reply:
@chrislovecnm I think `private-us-test-1a.privatecalico.example.com` is actually the correct name, that's how our calico route tables are currently named across the board.  

If that's correct, then the tests below are actually incorrect but that all depends on how detailed you want to be in these tests. It looks like the rest of the tests below do not specify the full subnet name, as the name tag, so maybe this is fine for testing with your change?

Reply:
I am wondering if the names are actually wrong ;(

Reply:
Let me be more clear. I am wondering if I came across a bug, as I am uncertain why the name is different from every other component.

Reply:
I believe they are incorrect / bug as well.  Any thoughts @robinpercy @blakebarnett?

Reply:
@mikesplain we may have to maintain two tags not certain.  We have to have the correct tag format:

```
 "kubernetes.io/cluster/$CLUSTERNAME" = "owned"
```
or
```
 "kubernetes.io/cluster/$CLUSTERNAME" = "shared"
```

These tags allow for not deleting shared resources, which is currently happening in our code :(

So maybe I leave the Name tag alone and just add the new tag `kubernetes.io/cluster/` correctly.  I think only the private route table is the only one where the `Name` tag is different.

Reply:
They certainly seem to be wrong in the tests, but yeah, it's just the name, not crucial.  Consistency is good though.